### ISSUE TITLE
Incluir campo email en el módelo Member

### ISSUE THREAD

**Issue by euribates** [pythoncanarias/pycan-web#371]
Se estaba usando el campo `email` del modelo `User`, pero esto presenta dos problemas que se nos pasaron por alto conado se vcreo el modelo.

- El campo `email` **no es obligatorio** en `auth.User`. Para nosotros es imprescindible que tengamos el correo del socio.
- Lo que es peor, `User.email` **no es único**.

Además, seria recomendable que, cuando un miembro cambien el email, se mantenga el registro de los email anteriores, por si hubiera un caso de robo de la cuenta (Muy poco problable, pero posible)

**Comment by jdevera** [comment]
>  Además, seria recomendable que, cuando un socio cambien el email, se mantenga el registro de los email anteriores, por si hubiera un caso de robo de la cuenta (Muy poco problable, pero posible).

Cuando alguien borra algo, deberíamos borrarlo. Si hay un robo de cuenta lo podemos resolver al estilo humano. No necesitamos escalar tanto a costa de berenjenales de datos de la gente, creo.

**Comment by sdelquin** [comment]
> > Además, seria recomendable que, cuando un socio cambien el email, se mantenga el registro de los email anteriores, por si hubiera un caso de robo de la cuenta (Muy poco problable, pero posible).
> 
> Cuando alguien borra algo, deberíamos borrarlo. Si hay un robo de cuenta lo podemos resolver al estilo humano. No necesitamos escalar tanto a costa de berenjenales de datos de la gente, creo.

Sí. Estoy de acuerdo con @jdevera en esto último. Me parece un poco de sobreingeniería
